index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/43fc886676.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Gaming Convention</title>
+  </head>
+  <body>
+    <div id="container">
+      <header class="head-section">
+        <div class="hamburger-icon">
+          <button class="hamburger-button" type="button">
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+          </button>
+        </div>
+        <div class="nav-bars">
+          <nav class="nav-bar-one">
+            <ul>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
```

Comment:
- [ ] Again here on the index page, kindly fix the issue with the whitespace around the top `nav`.

Your design
![image](https://user-images.githubusercontent.com/81932478/169434709-20791a9a-2e2d-4f3d-ab9e-26a62765fc57.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/169434756-d6ca0d61-3233-463d-a33c-211cee38f0f6.png)

Reply:
fixed

Reply:
Implemented 👍